templates/EC2_Quickstart.yml

```diff
@@ -0,0 +1,156 @@
+    Default: 'https://log-api.newrelic.com/log/v1'
+    AllowedValues:
+      - 'https://log-api.newrelic.com/log/v1'
+      - 'https://log-api.eu.newrelic.com/log/v1'
+Resources:
+
+  CTNewRelicInstallInfraAgentInstaller:
+    Description: Installs latest version NewRelic Infrastructure agent in EC2 instances once every month.
+    Type: 'AWS::SSM::Association'
+    Properties:
+      AssociationName: CT-NewRelic-Infra-Agent-Installer
+      Name: AWS-ConfigureAWSPackage
+      ScheduleExpression: "rate(30 days)"
+      Targets:
+        - Key: "tag:Supervisor"
+          Values:
+            - "new-relic"
+      Parameters:
+        action: 
+          - !Ref Action
+        installationType: 
+          - !Ref InstallationType
+        name: 
+          - !Ref NewRelicInfrastructureAgent
+        additionalArguments: 
+          - !Ref AdditionalParametersLicenseKey
+
+  NewRelicApiDestination:
+    DependsOn: NewRelicConnection
+    Type: AWS::Events::ApiDestination
+    Properties:
+      Name: CT-NewRelic-Logs-Endpoint
+      HttpMethod: POST
+      InvocationEndpoint: !Ref NewRelicLogsEndpoint
+      ConnectionArn: !GetAtt NewRelicConnection.Arn
+
+  NewRelicConnection:
+    Type: AWS::Events::Connection
+    Properties:
+      Name: CT-NewRelic-Endpoint-Connection
+      AuthorizationType: API_KEY
+      AuthParameters:
+        ApiKeyAuthParameters:
+          ApiKeyName: Api-Key
+          ApiKeyValue: !Ref NewRelicLicenseKey
+        InvocationHttpParameters: 
+          HeaderParameters: 
+            - Key: Api-Key
+              Value: !Ref NewRelicLicenseKey
+        
+  EventBridgeApiDestinationPolicy:
+    DependsOn: InvokeApiDestinationEventBridgeRole
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: CT-NewRelic-EventBridgeApiDestinationPolicy
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - "events:InvokeApiDestination"
+            Resource:
+              - !Sub 'arn:aws:events:${AWS::Region}:${AWS::AccountId}:api-destination/CT-Newrelic-Logs-Endpoint/*'
+      Roles:
+        - !Ref InvokeApiDestinationEventBridgeRole
+        
+  InvokeApiDestinationEventBridgeRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: CT-NewRelic-EventBridge-Invoke-Api-Destination
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: events.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonEventBridgeFullAccess
+    
+  SendLogsToNREndpoint:
+    Type: AWS::Events::Rule
+    Properties:
+      Name: CT-NewRelic-Failed-EC2-Infra-Agent-Installations-Logs-Rule
+      EventPattern:
+        source:
+          - "aws.ssm"
+        detail-type:
+          - "EC2 Command Invocation Status-change Notification"
+        detail:
+          status:
+            - "Failed"

```

Comment:
Can this be done for success as well ? AWS Console as an equivalent gives success and failure and IMO, a complementary dashboard at NEWR end would ensure that customers are able to monitor this successfully. Curious, if there are any other thought process to not do success statses ?